client/betterblend/src/Home.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './css/playlist.css';
+import axios from 'axios';
+
+class Home extends Component {
+
+	constructor() {
+		super();
+		this.state = {
+			classnames: '',
+			playlists: []
+		}
+	}
+
+	componentDidMount() {
+		this.getPlaylists();
+	}
+
+	getPlaylists = () => {
+		axios.get('/api')
+		.then(data => this.setState({playlists: data.data.playlists}));
+	};
+
+	render() {
+		return (
+			<div className={this.state.classnames}>
+				{this.state.playlists.map(p => 
```

Comment:
I've been trying to think of a better way to handle this because often the playlist may not have an image.
I'll put out a PR in a bit to show you my solution but I'm not a huge fan.
It's essentially a filter before the return statement.

Reply:
no worries I will remove that and make it into something else, but pics for tracks ok?

Reply:
do you have any ideas on how to handle that? i think we should merge first and do an iteration after